foundry/foundry.py

```diff
@@ -761,21 +721,21 @@ def _load_data(self, file=None, source_id=None, as_hdf5=False):
                     path_to_file
                 )
             except Exception as e:
-                print(f"Reading {file} as JSON failed: {e} \n", "Now attempting to read as JSONL")
+                logging.log(logging.INFO, f"Reading {file} as JSON failed: {e} \n", "Now attempting to read as JSONL")
                 try:
                     # Try to read individual lines instead
                     self.dataset.dataframe = pd.read_json(
                         path_to_file, lines=True
                     )
                 except Exception as f:
-                    print(f"Reading {file} as JSONL failed: {f} \n", "Now attempting to read as CSV")
+                    logging.log(logging.INFO, f"Reading {file} as JSONL failed: {f} \n", "Now attempting to read as CSV")
```

Comment:
See above comment about using a module-level logger.

Reply:
Sorry, not quite sure what you mean by "module" (is that function or class?)